components/hdw-tft/hdw-tft.c

```diff
@@ -377,7 +387,7 @@ void initTFT(display_t * disp, spi_host_device_t spiHost, gpio_num_t sclk,
 
 #if defined(CONFIG_ST7735_160x80)
     ESP_ERROR_CHECK(esp_lcd_new_panel_st7735(io_handle, &panel_config, &panel_handle));
-#elif defined(CONFIG_ST7789_240x135) || defined(CONFIG_ST7789_240x240)
+#elif defined(CONFIG_ST7789_240x135) || defined(CONFIG_ST7789_240x240) || defined(CONFIG_ST7735_128x160)
```

Comment:
Did you mean to put `CONFIG_ST7735_128x160` where it calls `esp_lcd_panel_st7789()` and not above where it calls `esp_lcd_panel_st7735()`?